csrankings-h.csv

```diff
@@ -966,6 +966,7 @@ Husheng Li,University of Tennessee,http://web.eecs.utk.edu/~husheng,pM3wc_YAAAAJ
 Hüsnü Yenigün,Sabancı University,http://people.sabanciuniv.edu/yenigun,VaDb-YYAAAAJ
 Hussein M. Alnuweiri,Texas A&M at Qatar,https://www.qatar.tamu.edu/programs/ecen/faculty-and-staff/duplicate-of-dr.-patrick-linke,NOSCHOLARPAGE
 Hussein Mouftah,University of Ottawa,http://www.site.uottawa.ca/~mouftah,33FmwbwAAAAJ
+Hussein Sibai,Washington University in St. Louis,https://husseinsibai.web.illinois.edu/,St5uCSwAAAAJ

```

Comment:
Tell this faculty member they need to update their web page!

Reply:
I've updated it to the school's link and will let him know.